src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -0,0 +1,2021 @@
+      "Parameters": {
+        "Scenario": "oltp_insert_T96_TB4_REC100000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "96",
+        "NumTables": "4",
+        "RecordCount": "100000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC5000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "5000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC10000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "10000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC50000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "50000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "SysbenchOLTPClientExecutor",
+      "Parameters": {
+        "Scenario": "oltp_insert_T152_TB4_REC100000",
+        "DatabaseName": "sbtest", 
+        "Role": "Client",
+        "Threads": "152",
+        "NumTables": "4",
+        "RecordCount": "100000",
+        "DurationSecs": "1800",
+        "Workload": "oltp_insert",
+        "PackageName": "sysbench"
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "LinuxPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallRequiredLinuxPackages",
+        "Packages": "mysql-server, make, automake, libtool, pkg-config",
+        "Packages-Apt": "libaio-dev, libmysqlclient-dev, libssl-dev",
+        "Packages-Dnf": "libaio-devel, mariadb-devel, openssl-devel",
+        "Packages-Yum": "libaio-devel, mariadb-devel, openssl-devel",
+        "Packages-Zypper": "libaio-dev, libmysqlclient-devel, openssl-devel"
+      }
+    },
+    {
+      "Type": "GitRepoClone",
+      "Parameters": {
+        "Scenario": "CloneSysbenchRepo",
+        "RepoUri": "https://github.com/akopytov/sysbench.git",
+        "PackageName": "sysbench"
+      }
+    },
+    {
+      "Type": "MySQLServerConfiguration",
+      "Parameters": {
+        "Scenario": "StartDatabaseServer",
+        "Action": "StartServer",
+        "DatabaseName": "$.Parameters.DatabaseName"
+      }
+    },
+    {
+      "Type": "MySQLServerConfiguration",
```

Comment:
For those dependency classes that is already here, please take a look just to make sure they are appropriate for server/client, for example I don't see a "role:server" in some of those sections.